lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts

```diff
@@ -87,4 +87,26 @@ export class LegalHoldService {
     unassignHold(holdId: string, holdChildId: string): Observable<undefined> {
         return from(this.legalHoldApi.unassignHold(holdId, holdChildId));
     }
+
+    /**
+     * Create hold.
+     *
+     * @param filePlanId The identifier of a file plan. You can also use the -filePlan- alias.
+     * @param holds Holds to create
+     * @returns List of created holds Observable<HoldEntry[]>
+     */
+    createHold(filePlanId: string, holds: Hold[]): Observable<HoldEntry> {
+        return from(this.legalHoldApi.createHold(filePlanId, holds));
+    }
+
+    /**
+     * Create list of holds.
+     *
+     * @param filePlanId The identifier of a file plan. You can also use the -filePlan- alias.
+     * @param holds Holds to create
+     * @returns List of created holds Observable<HoldPaging[]>
+     */
+    createHolds(filePlanId: string, holds: Hold[]): Observable<HoldPaging> {
```

Comment:
There is no point in having two separate methods you can instead modify the return type to something like `Observable<HoldPaging | HoldEntry>` there should be some examples already existing in the repo

Reply:
As we alined with you and @DenysVuika should be 2 separate methods